src/movegen/rooks.h

```diff
@@ -102,17 +119,20 @@ constexpr auto generateRookAttackTable()
 }
 
 constexpr auto s_rookAttackTable = generateRookAttackTable();
+} // end namespace
 
-}
-
-/* https://www.chessprogramming.org/Magic_Bitboards */
 static inline uint64_t getRookMoves(BoardPosition pos, uint64_t occupancy)
```

Comment:
This would probably be a small gain as well. (feel free to change if you like)
```suggestion
constexpr uint64_t getRookMoves(BoardPosition pos, uint64_t occupancy)
```